src/lib/gadgets/range-check.ts

```diff
@@ -0,0 +1,14 @@
+import { type Field } from '../field.js';
+import * as Gates from '../gates.js';
+
+export { rangeCheck64 };
+
+function rangeCheck64(x: Field) {
```

Comment:
Same comment from here: https://github.com/o1-labs/o1js/pull/1181#discussion_r1357115891

Reply:
will address this in the next PR, thanks!